### PR TITLE
Change CLIENT_ID to CLIENT_NAME

### PR DESCRIPTION
**Pull request by sedroche**
I believe we are going with `CLIENT_NAME` now so these are the changes for the CRC.

### CODE REVIEW

**Comment by sedroche** [comment]
@dimitraz @vchepeli

**Review by dimitraz — APPROVED** [review]
:+1:

File: roles/test-crc-apb/vars/main.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ SERVICE_CONFIG: |
           { "key" : 3 }
       ]
   }
-CLIENT_ID: 'MyClientId'
+CLIENT_NAME: 'MyClientId'
```
Comment (ghost) [inline roles/test-crc-apb/vars/main.yml]: should be `MyClientName`

**Review by ghost — CHANGES_REQUESTED** [review]

File: apb.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ plans:
       display_type: textarea
       title: Service JSON Configuration
     bind_parameters:
-    - name: CLIENT_ID
+    - name: CLIENT_NAME
       required: True
       type: string
       title: Mobile Client Id
```
Comment (ghost) [inline apb.yml]: `title: Mobile Client Name`

**Review by ghost — CHANGES_REQUESTED** [review]

File: roles/bind-crc-apb/tasks/main.yml [hunk 3]
```diff
@@ -35,7 +35,7 @@
       mobile: '{{ crc_svc_label_mobile }}'
       serviceName: '{{ crc_svc_name }}'
       serviceInstanceID: '{{ _apb_service_instance_id }}'
-      mobileClientId: '{{ CLIENT_ID }}'
+      mobileClientId: '{{ CLIENT_NAME }}'
```
Comment (ghost) [inline roles/bind-crc-apb/tasks/main.yml]: should be `clientId: '{{ CLIENT_NAME }}'`

File: roles/bind-crc-apb/tasks/main.yml [hunk 4]
```diff
@@ -35,7 +35,7 @@
...
       mobile-client: "enabled"
```
Comment (ghost) [inline roles/bind-crc-apb/tasks/main.yml]: should be  `mobile-client: '{{ crc_svc_label_mobile }}'`

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**